src/apprt/gtk/App.zig

```diff
@@ -1508,6 +1515,22 @@ pub fn quitNow(self: *App) void {
     self.running = false;
 }
 
+// SIGUSR2 signal handler via g_unix_signal_add
+fn sigusr2(ud: ?*anyopaque) callconv(.c) c_int {
+    const self: *App = @ptrCast(@alignCast(ud orelse
+        return @intFromBool(glib.SOURCE_CONTINUE)));
```

Comment:
Things are arguably extremely fucked if this triggers so we should maybe remove the source but I don't think it really matters either way.

Reply:
I agree, if this fails it doesn't matter because things are way messed up.